### ISSUE TITLE
uninitialized constant OpenTelemetry::Instrumentation::Base

### ISSUE THREAD

**Issue by PuntoDiGoccia** [open-telemetry/opentelemetry-ruby#300]
I'm trying auto-inspection on `rails`

`Gemfile`:
```
gem 'opentelemetry-api'
gem 'opentelemetry-sdk'
gem 'opentelemetry-exporters-jaeger'
gem 'opentelemetry-instrumentation-all'
```

`application.rb`:
```
require 'opentelemetry/api`
require 'opentelemetry/sdk'
require 'opentelemetry/exporters/jaeger'
require 'opentelemetry/instrumentation/all`
```

`bin/rails server` fails with the message in subject.
Looking at the stack, RubyGem gets confused: 
in 
`opentelemetry-api-0.5.0/lib/opentelemetry.rb`: `require 'opentelemetry/instrumentation'` (line 14)
loads 
`opentelemetry-instrumentation-sinatra-0.5.0/lib/opentelemetry/instrumentation.rb` 
instead of 
`opentelemetry-api-0.5.0/lib/opentelemetry/instrumentation.rb`

Possibly easiest solution is to change `require` -> `require_relative`

**Comment by fbogsany** [comment]
Fix in #301. Thanks for the report! I reproduced the failure in a small test app & verified the fix. Note that your first `require` should be `require 'opentelemetry'` rather than `require 'opentelemetry/api'`.

**Comment by mwear** [comment]
Fix released in https://rubygems.org/gems/opentelemetry-api/versions/0.5.1
